app/models/contactModel.go

```diff
@@ -0,0 +1,62 @@
+package models
+
+import (
+	"github.com/gocql/gocql"
+	"fmt"
+)
+
+type PhoneNum struct {
+	Id int
+	ContactId gocql.UUID
+	Phonenumber string
+
+}
+
+type Contact struct{
+	Id gocql.UUID
+	FirstName string
+	LastName string
+	Email string
+	PhoneNumbersStamped []PhoneNum
+	PhoneNumbers []string
+
+}
+
+
+func (contact  * Contact)StampContactId () {
```

Comment:
from code below you use this function to return ID of each PhoneNumber of a specific contact if what i say is right i think this function is unnecessary as you can do this in query of GetContacts unless there is modeling issue in Cassandra